pkg/logs/status/status.go

```diff
@@ -53,14 +53,14 @@ type Integration struct {
 
 // Status provides some information about logs-agent.
 type Status struct {
-	IsRunning        bool               `json:"is_running"`
-	Endpoints        []string           `json:"endpoints"`
-	StatusMetrics    map[string]int64   `json:"metrics"`
-	ProcessFileStats map[string]float64 `json:"process_file_stats"`
-	Integrations     []Integration      `json:"integrations"`
-	Errors           []string           `json:"errors"`
-	Warnings         []string           `json:"warnings"`
-	UseHTTP          bool               `json:"use_http"`
+	IsRunning        bool              `json:"is_running"`
+	Endpoints        []string          `json:"endpoints"`
+	StatusMetrics    map[string]int64  `json:"metrics"`
+	ProcessFileStats map[string]string `json:"process_file_stats"`
```

Comment:
Where is this JSON used? Changing the type of the field can break things.

Reply:
The JSON is used when the status command is called. The only struct type I'm changing is `ProcessFileStats`, which is a map that I added to the struct.